src/pages/trade-options/option-tabs/option-tabs.scss

```diff
@@ -0,0 +1,39 @@
+@import "../../../variables";
+
+.option-tabs{
```

Comment:
On mobile, the dates are moved to the top navigation bar, so we can hide the tabs IMO.

![image](https://user-images.githubusercontent.com/14966470/93195003-b0f76080-f783-11ea-857a-73b830a65229.png)